3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,117 @@
+//Age-ify (A future age calculator)
+
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
+console.log(`you will be in ${age} years old in ${yearFuture}`);
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
+ 
+
+let dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+
+let dogAgeInDogYears = dogAgeInHumanYears * 7;
+
+if (shouldShowResultInDogYears) {
+  console.log(`your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`);
+} else {
+  console.log(`your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`);
+}
+
+
```

Comment:
👌 Your distinction between dogAgeInHumanYears and dogAgeInDogYears adds clarity to the code. Well thought out!
🔍 Just a minor upgrade, if you are not going to update the variable, you can consider using `const`